qc/rnaseqQC.sh

```diff
@@ -1,8 +1,8 @@
 #!/usr/bin/env bash
 
 # Source script from the same (prod or test) Atlas environment as this script
-scriptDir=$(cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )
-projectRoot=${scriptDir}/../..
+#scriptDir=$(cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )
+scriptDir=$2
```

Comment:
I'm not sure if delegating this upstream might be a good idea...

Reply:
I think you are right @pcm32 I've reversed the change [9bad55c](https://github.com/ebi-gene-expression-group/atlas-analysis/pull/6/commits/9bad55c50493c7f58952fcbcf98729ac50f16158)